public/index.html

```diff
@@ -3,19 +3,17 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Page Title</title>
+    <title>Proxy Page</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="http://localhost:3016/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
 </head>
 <body>
-    <h1>Proxy Server</h1>
-    <div id="app"></div>
+    <div id="Details"></div>
+    <div id="events"></div>
     <script src="/lib/react.development.js"></script>
```

Comment:
Let's be consistent here.  I'd suggest making the id's Proper Case.  That way they are easily recognized. 

Also, I recall you using the word 'Categories' before.  I think that best explains what your component is and does.  It renders additional Categories of Attractions the purchaser may be interested in.

Reply:
Thanks. Done!